packages/runtime/container-runtime/src/opLifecycle/definitions.ts

```diff
@@ -41,4 +41,13 @@ export interface IChunkedOp {
     totalChunks: number;
     contents: string;
     originalType: MessageType | ContainerMessageType;
+    metadata?: Record<string, unknown>;
+    compression?: string;
+}
+
+type ProcessingState = "Processed" | "Skipped" | "Accepted";
```

Comment:
It's hard to say what these states means / track. Maybe add comments around all new types?
Also, I think you need to export it, given it's used in exported interface.

Reply:
Will add comments. Not sure why this needs to be exported. The interface is package private and TS doesn't complain about it.